### PR TITLE
Makes Pixel Perfect 2x the default pixel scaling method

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
For whatever reason, Stretch to fit was the default value for the pixel scaling method preference. That means that, by default, unless you were extremely lucky with your screen resolution, and/or fought a while with the width of your chat window, your game would oftentimes look disproportionate, and would lose a lot of pixel fidelity, rendering many things uglier than they should be.

**What's the difference between the two?**
Stretch to fit will stretch the width and height of the pixels displayed on your screen to match the entire window they're contained in. That means that, unless you have a perfect ratio for that window, you'll have certain rows or columns of pixels that are wider or narrower than others.
Pixel Perfect, on the other hand, will use the exact amount of pixels on your screen to match the size of an in-game pixel. That means that Pixel Perfect 2x will take a 2 by 2 square of pixels on your screen to display a game pixel, ensuring that every pixel has the same size.

**Why does it matter?**
/tg/ has been moving forward with massive improvements to its sprites in the past year or two, and it's a shame that new players don't get to fully enjoy that beauty because the resolution of the game doesn't do them justice.

## Why It's Good For The Game
New players should be able to choose to change their setting to Stretch to fit if they feel like that'd be an improvement for them, rather than making it the default. I think that pixel fidelity (and therefor, sprite fidelity) is more important than avoiding some black bars at both ends of your screen. It becomes a conscious decision to make things look unlike what they're intended to look like.

<details><summary>I mean, look at these:</summary>

Stretch to fit:
image

Pixel Perfect 2x:
image
</details>

<details><summary>Or these</summary>

Stretch to fit:
image
(notice the deformed right leg of the captain, that wider split on that one tile and that guy with one eye wider than the other)

Pixel Perfect 2x:
image
(all is uniform)
</details>

## Changelog

:cl: GoldenAlpharex
qol: Pixel Perfect 2x is now the default pixel scaling method, rather than Stretch to fit, so that new players will be able to enjoy the game's visuals as they're meant to look like, rather than oftentimes distorted.
/:cl:

### CODE REVIEW

**Comment by Ghilker** [comment]
image

**Comment by GoldenAlpharex** [comment]
Here, updated the description with clearer screenshots.

**Comment by Ghilker** [comment]
image

**Review by Imaginos16 — APPROVED** [review]
Please please please please PLEASE


The amount of people who use stretch to fit when taking screenshots in PRs disturb me. So if they can't put it simply on the settings, then I'd definitely want it to be the default so I don't have to deal with the issue on every review.

**Comment by MTandi** [comment]
2x looks terrible on 4k monitor, and I can't use 3x because the toolbar is obscured.

**Comment by Onule** [comment]
> 2x looks small on 4k monitor, and I can't use 3x because the toolbar is obscured.

This just makes it the default option, you can still change it to whatever you want.

**Comment by GoldenAlpharex** [comment]
> 2x looks small on 4k monitor, and I can't use 3x because the toolbar is obscured.
> 
> Is it actually possible to make the UI follow the borders regardless of the game viewport? I would use 3x if the equipment icons weren't cropped.

As Onule said, you can still use Stretch to fit if you want, this is just so the default value isn't Stretch to fit anymore. No need to worry about it ;)

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
Not a fan of this. Stretch to fit is defaut because it will work "ok" on all displays, even if it looks crummy for most.

If you want to get people to stop using the defaults, give them a new player settings rundown or something, so they can set these values as they prefer. Don't just change off the best all around value. S not a good pattern.

**Comment by wesoda25** [comment]
Add a tip of the round about it or something I had no clue you could fiddle with this

**Comment by NamelessFairy** [comment]
Echoing Lemons concern, have you had a chance to test this at different resolutions? Stretch to fit is effectively compatible with all displays while 2X seems to be better visually but with potentially worse compatibility. Mtandi has pointed out the issues with this on a 4k monitor and I'll echo their concerns on a QHD monitor, its certainly "playable" there's just significantly distracting black bars occupying about a 3rd of the gamespace that arguably provide a worse experience then slightly warped sprites. I'd also be interested to hear how this looks on smaller monitors, according to steams hardware hardware survey 900p and below monitors make up more than 10% of the market so its a non-ignorable amount.

TLDR: Agree with Lemon, 2X adds distracting black bars on a QHD display, how does this look on smaller monitors?

**Comment by Mothblocks** [comment]
Last I asked Lemon was giving you time to voice your opinion to that but you don't seem to be taking that opportunity
